nautobot_golden_config/nornir_plays/config_backup.py

```diff
@@ -93,8 +93,12 @@ def config_backup(job_result, data):
     """Nornir play to backup configurations."""
     now = datetime.now()
     logger = NornirLogger(__name__, job_result, data.get("debug"))
-    global_settings = GoldenConfigSetting.objects.first()
-    verify_global_settings(logger, global_settings, ["backup_path_template"])
+
+    qs = get_job_filter(data)
+    device_to_settings_map = get_device_to_settings_map(queryset=qs)
+
+    for settings in set(device_to_settings_map.values()):
```

Comment:
Why is `set` needed here? Each device should only be allowed to set one setting correct?

Reply:
it's possible to have multiple settings that should be  validated if the queryset generated by data includes devices that are mapped to different settings

Reply:
@jeffkala This is to validate setting objects only one time if multiple devices are re-using same 'settings' object.